src/views/certifications/components/standalone-link-contents/index.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import s from './standalone-link-contents.module.css'
+
+interface StandaloneLinkContentsProps {
+	text: string
+	size?: 'small' | 'medium'
+}
+
+/**
+ * A text element that looks like a standalone link,
+ * designed for use in CardLink.
+ *
+ * Note that there is not hover effect for this component.
```

Comment:
💭 Could we add a breadcrumb linking to the component that does use the hover (I assume there is one)?

Reply:
✅ added a clearer mention of `src/components/try-hcp-callout/components/standalone-link-contents`